build/root/Makefile.generated_files

```diff
@@ -402,6 +402,10 @@ $(CONVERSION_GEN): $(k8s.io/kubernetes/vendor/k8s.io/code-generator/cmd/conversi
 OPENAPI_BASENAME := $(GENERATED_FILE_PREFIX)openapi
 OPENAPI_FILENAME := $(OPENAPI_BASENAME).go
 OPENAPI_OUTPUT_PKG := pkg/generated/openapi
+BOILERPLATE_FILENAME := vendor/k8s.io/code-generator/hack/boilerplate.go.txt
+REPORT_FILENAME := $(OUT_DIR)/violations.report
+KNOWN_VIOLATION_FILENAME := hack/testdata/api-rules/violations.report
```

Comment:
should this be called `violation_exceptions.list" or something like that? (change list to the format of the file, csv, yaml, etc?)

Reply:
Yes it's indeed an exception / whitelist file. I used "violations.report" instead of "whitelist" because I didn't want people to feel that it's okay to add more entries to it. 

I updated the README to emphasize that the entries in the list should only be removed but not added. Since the file is not in csv format, I updated the filename to `violation_exceptions.list`